### ISSUE TITLE
[Bug]: The value of role is causing Fast Pass to fail.

### ISSUE THREAD

**Issue by mehulnirala** [microsoft/fluentui#22929]
### Environment Information

- **Package version(s)**:  7.183.1
- **Browser and OS versions**: (fill this out if relevant)
OS: Windows 11
Browser: Microsoft edge

### Describe the issue:
FastPass is failing due to the value of role in Suggestions Items
![image](https://user-images.githubusercontent.com/86864820/167787742-55820ba8-c941-4990-a173-e3937c49465a.png)

![image](https://user-images.githubusercontent.com/86864820/167787203-13ed578a-4290-4ee7-9de4-c89f572edfb4.png)

#### Actual behavior:
role is set to `presentation`  in [Suggestionsitems.tsx](https://github.com/microsoft/fluentui/blob/7d9180debc95b08349c4df48b854c1bf5fe2dd46/packages/react/src/components/pickers/Suggestions/SuggestionsItem.tsx#L70)

#### Expected behavior:
role property should be set to `group`, `listbox`.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
@mehulnirala -  Thank you for filing this item with us. I'll add this to our contributors queue as it’s low on our priority list and we’re likely not going to get to it for a while as we’re currently focused on larger coherence work items and enhancements to our performance. 
In the meantime, if you want to contribute towards this request, please share your PR and our dev team will review it.
